### PR TITLE
Update microsoft-defender-offline.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated information on how to find scan results

Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9549

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/microsoft-defender-offline.md [hunk 1]
```diff
@@ -137,6 +137,14 @@ See the following for more information:
 > - [Configure Defender for Endpoint on Android features](android-configure.md)
 > - [Configure Microsoft Defender for Endpoint on iOS features](ios-configure-features.md)
 
+### Where can I find scan results?
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-offline.md]:
```suggestion
### Where can I find the scan results?
```

File: microsoft-365/security/defender-endpoint/microsoft-defender-offline.md [hunk 2]
```diff
@@ -137,6 +137,14 @@ See the following for more information:
...
+
+To see the Microsoft Defender Offline scan results:
+
+Select **Start** , and then select **Settings**  > **Update & Security**  > **Windows Security**  > **Virus & threat protection** .
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-offline.md]:
```suggestion
1. Select **Start**, and then select **Settings**  > **Update & Security**  > **Windows Security**  > **Virus & threat protection**.
```

File: microsoft-365/security/defender-endpoint/microsoft-defender-offline.md [hunk 3]
```diff
@@ -137,6 +137,14 @@ See the following for more information:
...
+
+On the **Virus & threat protection** screen, under **Current threats**, select **Scan** options, and then select **Protection history** .
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-offline.md]:
```suggestion
2. On the **Virus & threat protection** screen, under **Current threats**, select **Scan options**, and then select **Protection history**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by denisebmsft — APPROVED** [review]
Thank you so much @msbemba @JohanFreelancer9! I appreciate your help!

**Pull request merged**